docs/source/noise/column_noise.rst

```diff
@@ -4,18 +4,128 @@
+The types implemented in pseudopeople are listed below, along with instructions
+about how to configure them.
+
+.. contents::
+   :depth: 2
+   :local:
+
 Leave a field blank
 -------------------
 
+When this noise type occurs, whatever value was supposed to be collected is missing for this row.
+This could be because the input was left blank, an answer was refused,
+or the answer was illegible or unintelligible.
+pseudopeople will output a :code:`numpy.nan` value in
+the relevant cell.
+
+This noise type is called :code:`leave_blank` in the configuration. It takes one parameter:
+
+.. list-table:: Parameters to the :code:`leave_blank` noise type
+  :widths: 1 5 1
+  :header-rows: 1
+
+  * - Parameter
+    - Description
+    - Default
+  * - :code:`cell_probability`
+    - The probability that a cell in the column being configured is blank.
+      This probability is the same for all cells in a column;
+      missingness is not correlated with any attributes.
+    - 0.01 (1%)
+
 Choose the wrong option
 -----------------------
 
-Write the wrong zipcode digits
-------------------------------
+Some survey questions or administrative forms ask the respondent to choose
+between a list of options.
+This noise type occurs when the wrong option is chosen by mistake.
+The options available depend on the column; see the :ref:`datasets page <datasets_main>` for
+lists of options that correspond to each column.
+All wrong options are equally likely.
+
+This noise type is called :code:`choose_wrong_option` in the configuration.
+It takes one parameter:
+
+.. list-table:: Parameters to the :code:`choose_wrong_option` noise type
+  :widths: 1 5 1
+  :header-rows: 1
+
+  * - Parameter
+    - Description
+    - Default
+  * - :code:`cell_probability`
+    - The probability that, for a cell in the column being configured, the wrong option is chosen.
+      The probability of this noise type is the same for all cells in a column;
+      it is not correlated with any attributes.
+    - 0.01 (1%)
+
+Write the wrong ZIP code digits
+-------------------------------
+
+When reporting a ZIP code on a survey or form, people may misremember or misreport
+the digits.
+They are probably more likely to get confused about the last few digits (which identify
+the small, specific area) than the first few (which will be the same over a larger area).
+
+This noise type is called :code:`write_wrong_zipcode_digits` in the configuration.
+It takes two parameters:
+
+.. list-table:: Parameters to the :code:`write_wrong_zipcode_digits` noise type
+  :widths: 1 5 1
+  :header-rows: 1
+
+  * - Parameter
+    - Description
+    - Default
+  * - :code:`cell_probability`
+    - The probability of a cell being *considered* to have this noise type.
+      One way to think about this is the probability that a ZIP code is reported by someone who isn't sure of their ZIP code.
+      This probability is the same for all cells in a column; it is not correlated with any attributes.
```

Comment:
Again, I'm not sure what you mean by "attributes."

Reply:
```suggestion
      This probability is the same for all cells in a column; it is not correlated with any simulant attributes.
```

Reply:
Yeah the issue here is that not all columns are _simulant_ attributes -- for example, the `event_date` column of SSA is about the event, not the simulant.

It seems clear that trying to say this concisely in each noise type's section is not working. I'll expand it and move it to the top of the page before the individual noise types.